aclk/agent_cloud_link.c

```diff
@@ -1391,6 +1392,7 @@ void *aclk_main(void *ptr)
         RSA_free(aclk_private_key);
 
     aclk_main_cleanup(ptr);
+    static_thread->enabled = NETDATA_MAIN_THREAD_EXITED;
```

Comment:
The `aclk_main_cleanup` will set this before returning (`NETDATA_MAIN_THREAD_EXITING` and then `NETDATA_MAIN_THREAD_EXITED`) so it can be skipped

Reply:
I would probably remove it from `aclk_main_cleanup` actually, and put `goto cleanup` for repeating 2 lines in code:
```
static_thread->enabled = NETDATA_MAIN_THREAD_EXITED;
    return NULL;
```

Reply:
True, will remove.